Obstacle.py

```diff
@@ -0,0 +1,3 @@
+class Obstacle:
+     def __init__(self):
```

Comment:
constructor needs to take several parameters such as the x,y position of the obstacle